Sources/Server/server.swift

```diff
@@ -62,7 +54,7 @@ public class Server {
             return parsedRequest
         } catch let error {
             print (error.localizedDescription)
-            return HttpRequest(method: "", url: "", version: "", headers: ["" : ""])
+            return HttpRequest(method: "", url: "", version: "", headers: ["" : ""], body: ["": ""])
```

Comment:
It might be nice to have a method called "emptyHeaders" that returns a `["" : ""]`. That would be a more descriptive term. Same for body maybe?